Cluster Mode Tutorial (#1147)
spectrocloud/librarium: content/docs/18-knowledgebase/01-tutorials/3-deploy-k8s-cluster.md

---
title: "Deploy a Kubernetes Cluster with Palette"
metaTitle: "Deploy a Kubernetes Cluster with Palette"
metaDescription: "Learn how to deploy an application on a Kubernetes cluster with Palette. Experience a streamlined approach to creating and managing multiple Kubernetes clusters, on different public cloud providers, through Palette's optimized process."
icon: ""
hideToC: false
fullWidth: false
hideToCSidebar: false
hiddenFromNav: false
---

import Tabs from 'shared/components/ui/Tabs';
import WarningBox from 'shared/components/WarningBox';
import InfoBox from 'shared/components/InfoBox';
import PointsOfInterest from 'shared/components/common/PointOfInterest';

# Deploy a Kubernetes Cluster with Palette

> Palette's purpose is to allow you to creating and managing a Kubernetes host cluster to the public cloud providers with minimal effort. Palette makes possibile for software engineers, application developers, or system administrators that want to deploy a containerized application, to interact with Kubernetes clusters with ease.

Reviewer: ```suggestion
Palette allows you to create and manage a Kubernetes host cluster in all major public cloud providers with minimal effort. Palette makes possible for software engineers, application developers, or system administrators that want to deploy a containerized application to interact with Kubernetes clusters with ease.
```
Creating suggestion to test Netlify integration.
